{{cookiecutter.repo_slug}}/README.md

```diff
@@ -152,20 +152,20 @@ See the [open issues](https://github.com/{{cookiecutter.github_username}}/{{cook
 Reach out to the maintainer at one of the following places:
 
 {% if cookiecutter.use_github_discussions == 'y' -%}
-- [GitHub discussions](https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.repo_slug}}/discussions)
+- [GitHub Discussions](https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.repo_slug}}/discussions)
```

Comment:
My personal stylistic thing: Discussions are a GitHub-specific thing, so in my opinion GitHub Discussions should be capitalized